astropy/nddata/nddata.py

```diff
@@ -140,32 +139,32 @@ def __init__(self, data, uncertainty=None, mask=None, wcs=None,
             # other parameters.
             if (unit is not None and data.unit is not None and
                     unit != data.unit):
-                log.info("Overwriting NDData's current "
-                         "unit with specified unit")
+                log.info("overwriting NDData's current "
```

Comment:
Well they mentioned in your link: "if in doubt, it might be safer to use the lower case."